### PR TITLE
Only show total user's credit in the current wallet

### PR DESCRIPTION
**Pull request by jsaq007**
## What do these changes do?

Screenshot 2023-11-20 at 09 28 46

## Related issue/s

- Resolves Show only total user credits #5034

## Dev Checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by sanderegg — APPROVED** [review]
great! don't forget to set the milestone, labels etc etc...

**Review by elisabettai — APPROVED** [review]
Than you, looks much better!

File: services/static-webserver/client/source/class/osparc/navigation/CreditsMenuButton.js [hunk 1]
```diff
@@ -108,24 +108,28 @@ qx.Class.define("osparc.navigation.CreditsMenuButton", {
...
+        //   used = currentUsage.getUsedCredits();
+        // }
         const creditsLeft = wallet.getCreditsAvailable();
         if (creditsLeft !== null) {
           text = "<span style='font-size:12px;display:inline-block'>CREDITS</span><br>";
           let nCreditsText = "";
-          if (used !== null) {
-            nCreditsText += osparc.desktop.credits.Utils.creditsToFixed(used) + " / ";
-          }
+          // if (used !== null) {
```
Comment (pcrespov) [inline services/static-webserver/client/source/class/osparc/navigation/CreditsMenuButton.js:124]: Our policy is not to version commented code.

**Review by pcrespov — CHANGES_REQUESTED** [review]
Please consider removing commented code

**Review by GitHK — APPROVED** [review]

File: services/static-webserver/client/source/class/osparc/navigation/CreditsMenuButton.js [hunk 2]
```diff
@@ -97,35 +96,18 @@ qx.Class.define("osparc.navigation.CreditsMenuButton", {
         wallet.addListener("changeCreditsAvailable", () => this.__updateCredits());
       }
     },
-    // Note - Sometimes it doesn't work and it bring the credits indicator into a failed state
```
Comment (pcrespov) [inline services/static-webserver/client/source/class/osparc/navigation/CreditsMenuButton.js:100]: @jsaq007 @ignapas we should not have commented code under vc

**Review by ignapas — APPROVED** [review]

**Pull request merged**